scripts/pluto-ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd pluto-message-ingestion
+
+# The pluto lambda uses a different version of node from main app
+# so we run it separately
+
+yarn
```

Comment:
You'll want to install dependencies in CI with `--frozen-lockfile` to ensure a deterministic build.

Similarly in `scripts/app-ci.sh` above.

From https://classic.yarnpkg.com/en/docs/cli/install/:

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass `--frozen-lockfile` flag.

Reply:
This wasn't present in the original scripts, we'll do this in a new pr.